doc/release_notes_0.7.rst

```diff
@@ -0,0 +1,382 @@
+- :acrn-issue:`1536` - dm: add virtio_mei mediator
+- :acrn-issue:`1544` - dm: rpmb: Support RPMB mode config from launch.sh
+- :acrn-issue:`1812` - export kdf_sha256 interface from crypto lib
+- :acrn-issue:`1815` - improve emulation of I/O port CF9
+- :acrn-issue:`1915` - Enable Audio Mediator
+- :acrn-issue:`1953` - Add cmdline option to disable/enable vhm module for guest
+- :acrn-issue:`2176` - Fix RTC issues in ACPI
+- :acrn-issue:`2319` - Add vHPET support
+- :acrn-issue:`2351` - Enable post-launched hybrid mode
+- :acrn-issue:`2407` - Coding style changes for IOAPIC and MSI representation
+- :acrn-issue:`2426` - Enable Interrupt Remapping feature
+- :acrn-issue:`2431` - VPCI code cleanup
+- :acrn-issue:`2448` - Adding support for socket as a backend for Virtio-Console
+- :acrn-issue:`2462` - Enable cache QOS with CAT
+- :acrn-issue:`2496` - VTD/IOMMU Modulization
+- :acrn-issue:`2553` - Disable vTPM feature on ACRN GP2.0(apl_sdc_stable)
+- :acrn-issue:`2595` - Update vsbl to v1.2
+
+Fixed Issues Details
+********************
+
+- :acrn-issue:`1319` - SD card pass-through: UOS can't see SD card after UOS reboot.
+- :acrn-issue:`1774` - UOS cannot stop by command: acrnctl stop [vm name] in SOS
+- :acrn-issue:`1780` - Some video formats cannot be played in SOS.
+- :acrn-issue:`1782` - UOS failed to get ip with the pass-throughed network card
+- :acrn-issue:`1999` - [APLNUC][KBLNUC][APLUP2]UOS reset fails with acrnctl reset command
+- :acrn-issue:`2276` - OVMF failed to launch UOS on UP2
+- :acrn-issue:`2298` - Hardcodes path to iasl
+- :acrn-issue:`2316` - Tools don't respect CFLAGS/LDFLAGS from environment
+- :acrn-issue:`2338` - [UP2]Lost 2G memory in SOS when using SBL as bootloader on UP2
+- :acrn-issue:`2370` - Doesn't use parallel make in subbuilds
+- :acrn-issue:`2422` - [PATCH] profiling: fix the system freeze issue when running profiling  tool
+- :acrn-issue:`2453` - Fix vHPET memory leak on device reset
+- :acrn-issue:`2455` - host call stack disappear when dumping
+- :acrn-issue:`2516` - [UP2][SBL] System hang with DP monitor connected
+- :acrn-issue:`2528` - [APLUP2] SBL (built by SBL latest code) failed to boot ACRN hypervisor
+- :acrn-issue:`2543` - vLAPIC: DCR not properly initialized
+- :acrn-issue:`2548` - [APLNUC/KBLNUC][GVT][SOS/LAAG] Weston fails to play video in SOS and UOS
+- :acrn-issue:`2572` - Startup SOS Fails
+- :acrn-issue:`2588` - Uninitialized Variable is used in acrn_kernel/drivers/acrn/acrn_trace.c and acrn_hvlog.c
+- :acrn-issue:`2606` - HV crash during running VMM related Hypercall fuzzing test.
+- :acrn-issue:`2624` - Loading PCI devices with table_count > CONFIG_MAX_MSIX_TABLE_NUM leads to writing outside of struct.
+- :acrn-issue:`2643` - Ethernet pass-through, network card can't get ip in uos
+- :acrn-issue:`2674` - VGPU needs the lock when updating ppggt/ggtt to avoid the race condition
+- :acrn-issue:`2695` - UOS powers off or suspend while pressing power key, UOS has no response
+
+Known Issues
```

Comment:
What is the difference betwen "New Issues" and "Known Issues"

Reply:
Not sure.  I'll just merge these two lists I got from Nanlin

Reply:
@dbkinder @gvancuts New feature list is the issues with type "feature/story", and know issues are with the type "bug". Suggest to split these two list as previous release did.

Reply:
OK, I separated the lists as they were.